### PR TITLE
Implement initial DB setup

### PR DESCRIPTION
**Pull request by jo-jstrm**

### CODE REVIEW

File: device-identifier/requirements.txt [hunk 1]
```diff
@@ -8,4 +8,7 @@ pyfiglet==0.8.post1
 loguru==0.6.0
 influxdb-client==1.30.0
 pyinstaller==5.1
-pandas==1.4.3
\ No newline at end of file
+pandas==1.4.3
+shbdeviceidentifier>=0.0.0
+requests>=2.28.1
+influxdb>=5.3.1
```
Comment (TB-DevAcc) [inline device-identifier/requirements.txt:14]: influxdb will have to be removed in future fixes.

File: device-identifier/shbdeviceidentifier/app.py [hunk 2]
```diff
@@ -98,17 +96,22 @@ def app(ctx, debug, silent, verbose, version_flag):
...
+@pass_ctx
+def start(ctx):
     """
     Starts the RPC server.
     """
-    run_rpc_server()
+    try:
+        run_rpc_server()
+    except KeyboardInterrupt:
+        logger.error("Stopping InfluxDB...")
```
Comment (TB-DevAcc) [inline device-identifier/shbdeviceidentifier/app.py:107]: this should be info or debug, not error

File: device-identifier/shbdeviceidentifier/db.py [hunk 3]
```diff
@@ -154,11 +248,6 @@ def _setup_SQLite_db(self):
                                                id integer PRIMARY KEY,
                                                username VARCHAR NOT NULL UNIQUE
                                            );"""
-
-        sql_add_user = """INSERT OR IGNORE INTO users (username)
```
Comment (TB-DevAcc) [inline device-identifier/shbdeviceidentifier/db.py:158]: Why is this part missing? When I test I cannot find any users in `users` anymore.

**Review by TB-DevAcc — CHANGES_REQUESTED** [review]
Additionally, we can now remove bash commands in cli_setup.txt, right?

File: device-identifier/shbdeviceidentifier/db.py (hunk 3 above)
Comment (jo-jstrm) [reply device-identifier/shbdeviceidentifier/db.py:158]: Fixed

File: device-identifier/requirements.txt (hunk 1 above)
Comment (jo-jstrm) [reply device-identifier/requirements.txt:14]: Why? We are using `influxdb.InfluxDBClient()`.

File: device-identifier/shbdeviceidentifier/app.py (hunk 2 above)
Comment (jo-jstrm) [reply device-identifier/shbdeviceidentifier/app.py]: Fixed

**Comment by jo-jstrm** [comment]
> Additionally, we can now remove bash commands in cli_setup.txt, right?

Yes!

File: device-identifier/requirements.txt (hunk 1 above)
Comment (TB-DevAcc) [reply device-identifier/requirements.txt:14]: Yes, but that is the old client (as you mentioned :D), so it should be removed. Not necessary now, just a reminder really. I'm resolving this for now.

**Review by TB-DevAcc — APPROVED** [review]
👍

**Pull request merged**